pitop/labs/web/responses.py

```diff
@@ -0,0 +1,36 @@
+import gevent
+from io import BytesIO
+from flask import Response
+from werkzeug.exceptions import HTTPException
+
+
+class VideoResponse(Response):
+    def __init__(self, camera=None, *args, **kwargs):
+        if camera is None:
+            raise HTTPException(409)
```

Comment:
AFAICT these highly specific http error codes are too unpopular to be considered best practice 🫖 (unsupported teapot emoji)

Reply:
lol

Reply:
Yeah, I think this isn't really a 400 or even 500 class error, the server should just crash if you create a VideoResponse without a camera probably. Either way I don't think this is working correctly, when I tried to test the server swallowed the errors and the client gets a 200 with no image data

Reply:
oh really? it was working for me, I'll have another look.